opentaskspal/src/main/java/org/dmfs/android/contentpal/testing/Check.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.testing;
+
+/**
+ * // TODO review javadocs
+ * Represents a check/matcher/condition/verification that can be checked against a target object of type <code>T</code>.
+ *
+ * @author Gabor Keszthelyi
+ */
+public interface Check<T>
```

Comment:
Do we really need this interface at all?

Reply:
Since we basically only have a single and simple `Matcher` now I suppose we don't, so I've removed it. I still think that it may be nice to have a custom flowable 'api' to test operations, but it's not important, we could get back to it later as well.